src/aiida_quantumespresso/calculations/neb.py

```diff
@@ -165,13 +173,13 @@ def validate_inputs(cls, inputs, _):
             # Check that a pseudo potential was specified for each kind present in the `StructureData`
             # self.inputs.pw.pseudos is a plumpy.utils.AttributesFrozendict
             kindnames = [kind.name for kind in structure_list[0].kinds]
-            if set(kindnames) != set(inputs['pw']['pseudos'].keys()):
-                formatted_pseudos = ', '.join(list(inputs['pw']['pseudos'].keys()))
+            if set(kindnames) != set(value['pw']['pseudos'].keys()):
+                formatted_pseudos = ', '.join(list(value['pw']['pseudos'].keys()))
                 formatted_kinds = ', '.join(list(kindnames))
                 return 'Mismatch between the defined pseudos and the list of kinds of the structure.\n' \
                        f'Pseudos: {formatted_pseudos};\nKinds: {formatted_kinds}'
 
-        cls.inputs = AttributeDict(inputs)
+        cls.inputs = AttributeDict(value)
```

Comment:
Side tangent: I didn't notice when reviewing that the inputs are actually updated. Within the "pydantic" definition of validation, I'm perfectly fine with this, also see [this comment](https://github.com/aiida-vasp/aiida-vasp/issues/799#issuecomment-3173028399).
>  **Validation**: The process of checking that data conforms to expected rules, formats, or constraints before using or storing it.
    **Serialization**: The process of converting Python objects into a format that can be stored or transmitted and later reconstructed (deserialization).

Besides, I'll be removing the old inputs soon for v5.0. Reminds me I should add these deprecations to the `v4.X` support branch. ^^